source/components/molecules/CaseCard/CaseCard.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Icon, Text } from '../../atoms';
+import { Card } from '..';
+
+Card.LargeText = styled(Card.Text)`
+  font-size: ${(props) => props.theme.fontSizes[4]}px;
+  font-weight: ${(props) => props.theme.fontWeights[1]};
+  margin-bottom: 4px;
+`;
+
+Card.Meta = styled(Card.Text)`
+  font-size: ${(props) => props.theme.fontSizes[3]}px;
+  ${(props) => `color: ${props.theme.colors.neutrals[1]};`}
+`;
+
+const CaseCard = ({
+  colorSchema,
+  icon,
+  name,
+  largeSubtitle,
+  subtitle,
+  description,
+  onCardClick,
+  showProgress,
+  currentStep,
+  totalSteps,
+  showButton,
+  buttonText,
+  buttonIconName,
+  onButtonClick,
+  showPayments,
+  approvedAmount,
+  declinedAmount,
+  givedate,
+}) => (
+  <Card colorSchema={colorSchema}>
+    <Card.Body shadow color="neutral" onPress={onCardClick}>
+      {icon && <Card.Image source={icon} />}
+      <Card.Title colorSchema="neutral">{name}</Card.Title>
```

Comment:
title seems like a more appropriate prop instead of name? Or am i missing something? :D

Reply:
No you're right, `title` is definitely better; I just took the existing variable names (from a case object) from the previous code that this is based on 🙂.